src/content/protocols/eigenlayer/ethereum.md

```diff
@@ -0,0 +1,624 @@
+      },
+      { text: "Contracts are immutable.", status: "fixed" },
+      { text: "Alternative third-party frontends exist", status: "fixed" },
+    ],
+  ]
+---
+
+# Summary
+
+Eigenlayer is a marketplace that matches compute (by _Operators_) with tasks (by _AVSs_). In order that _AVSs_ trust the _Operators_, the _Operators_ need economic security (stake), which is provided by _(Re)Stakers_, that can be slashed in case of shortcoming of operational performance. _(Re)Stakers_ have to trust _Operators_ that they do their tasks according to the requirements stated by the _AVSs_. For economic stake, _(Re)Stakers_ can use a) arbitrary ERC20 tokens, which also includes LSTs, or b) Beacon Chain deposits to increase capital efficiency of staked ETH.
+
+# Ratings
+
+## Chain
+
+The report is concerned with the Eigenlayer protocol deployed on Ethereum mainnet. Ethereum mainnet achieves a _Low_ centralization risk score.
+
+> Chain score: Low
+
+## Upgradeability
+
+Contracts which control user funds, the accounting of delegation and slashing are fully _upgradeable_ and this can lead to _loss of user funds_ and _loss of unclaimed yield_.
+
+The withdrawal process can be paused immediately by pausers registered in the `PauserRegistry` contract. Pausing can lead to temporary loss of access to funds, the [Community Council (9/13)](#security-council) however can resume the pause immediately if misused.
+
+Rewards committed by _AVSs_ to the `RewardsCoordinator` contract are distributed to _Operators_ and _Stakers_ via a merkle tree root posted by the `RewardsUpdater` role owner, which computes eligibility for each _(Re)Staker_ and _Operator_ based on their stake and _AVS_ rewards specification. If the account that holds the `RewardsUpdater` role is compromised, the rewards can be distributed to the wrong accounts, which leads to _loss of unclaimed yield_.
+
+> Upgradeability score: High
+
+## Autonomy
+
+The Eigenlayer protocol does not rely on a single _Operator_, nor on a single _AVS_ to work since it is a market place that brings _AVSs_ and _Operators_ together. Thus the overall _Dependency_ risk is _low_. However, single _AVSs_ or _Operators_ can impose risks to each other as the respective counterparties.
+
+The [Dependencies](#dependencies) section goes into more detail about risks potentially originating from _AVSs_ and _Operators_.
+
+> Autonomy score: Low
+
+## Exit Window
+
+### Upgrading Contracts
+
+Upgrading of all upgradeable Eigenlayer smart contracts can be executed through the [Executor Multisig (1/2)](#security-council) which has two signers, the [Community Multisig (9/13)](#security-council) and the [Protocol TimelockController](#security-council). The [Protocol TimelockController](#security-council) has an _Exit Window_ of 10 days. The upgrade can also be executed immediately through the [Community Council (9/13)](#security-council) signer which adheres to the [Security Council Requirements](/learn-more#security-council-requirements).
```

Comment:
Are there different delays between the executive multisig and the community one? Why mention the latter twice?

Reply:
yes they have different delayes (none vs 10 days and none vs 24 days)
[7919123](https://github.com/deficollective/defiscan/pull/188/commits/7919123368d836780b8d3209e2a0bd71d3a0eb44)